db/data.py

```diff
@@ -58,6 +58,52 @@ def _has_key(dictionary, key):
         dictionary = dictionary[part]
     return True
 
+def show_failed_rows():
+    """
+        checks for rows in highlevel table which havent been processed.
+        Returns:
+        (id, mbid, offset) of unprocessed rows sorted by submission date
+        where
+        1. Id : id of the failed row in highlevel table, 
+        2. Mbid : Musicbrainz Identifier, 
+        3. Offset : It is the position of the failed row in a set of rows
+                    having same mbids  
+    """
+    with db.engine.begin() as connection:
+        query = text("""
+                        SELECT id,mbid,
+                    ROW_NUMBER ()
+                          OVER (PARTITION BY mbid) RowNum
+                          FROM highlevel
+                     LEFT JOIN highlevel_meta
+                         USING (id)
+                         WHERE highlevel_meta.id is null
+                      ORDER BY submitted
+                    """)
+        result = connection.execute(query).fetchall()
+    return result
```

Comment:
we generally tend to return dicts instead of sqlalchemy resultproxy objects, I think we should do that here too.